diff --git a/pkg/enterprise/model_rbac_manager_role_member.go b/pkg/enterprise/model_rbac_manager_role_member.go
index 65795c7..8539ced 100644
--- a/pkg/enterprise/model_rbac_manager_role_member.go
+++ b/pkg/enterprise/model_rbac_manager_role_member.go
@@ -15,6 +15,7 @@ import (
 	"encoding/json"
 	"bytes"
 	"fmt"
+	"time"
 )
 
 // checks if the RbacManagerRoleMember type satisfies the MappedNullable interface at compile time
